src/Models/ElementalArea.php

```diff
@@ -193,8 +195,10 @@ public function getOwnerPage()
         }
 
         // Allow for repeated calls to read from cache
-        if (isset($this->cacheData['owner_page'])) {
-            return $this->cacheData['owner_page'];
+        $cacheKey = 'owner_page_'. Versioned::get_reading_mode();
+
```

Comment:
```suggestion
```

Reply:
I think an empty line here is ok, I personally dislike variable assignments glued together to code blocks below.

Reply:
I like them when the assignment is explicitly there to be used in the code block - ties it all together. But I'm okay keeping it there if the other change isn't being made. Bit of a (subjective) nitpick on my part.